### PR TITLE
Modify transaction db structure to match what BB need

### PR DESCRIPTION
**Pull request by cfagaras**
Change the transaction table structure in the database to have the fields needed in BB integration specs

### CODE REVIEW

File: step_impl/step_impl.py [hunk 1]
```diff
@@ -23,21 +24,64 @@ def random_amount():
...
+
 @step("Add a transaction every <frequency> seconds for <duration> seconds")
 def add_transactions(frequency, duration):
     print(
         f"Adding a transaction every {frequency} seconds for {duration} seconds . . ."
     )
-    sql = """INSERT INTO transactions(amount, time)
-             VALUES(%s, now()) RETURNING id;"""
+
+    # sql = """INSERT INTO transactions(amount, time)
```
Comment (johnboyes) [inline step_impl/step_impl.py:61]: This code [should be deleted, not commented out](https://blog.submain.com/delete-commented-code-without-reading/).

File: step_impl/step_impl.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+from doctest import debug
```
Comment (johnboyes) [inline step_impl/step_impl.py:1]: This comment should be removed

**Comment by cfagaras** [comment]
I deleted client table from dbchangelog.xml
I deleted the commented lines from spec impl.
I removed unused imports.

**Review by johnboyes — APPROVED** [review]
I've integration tested this with Kafka on [Confluent Cloud][1], and verified that it streams the transactions successfully into a Kafka topic using both the [PostgreSQL Source JDBC Connector][2], and also our target use case [PostgreSQL CDC Source Connector (Debezium)][3]. 😎 

Nice work @cfagaras 💥 

LGTM :shipit: 

[1]: https://www.confluent.io/confluent-cloud/
[2]: https://docs.confluent.io/cloud/current/connectors/cc-postgresql-source.html#postgresql-source-jdbc-connector-for-ccloud
[3]: https://docs.confluent.io/cloud/current/connectors/cc-postgresql-cdc-source-debezium.html

**Pull request merged**